src/cart/crt_iv.c

```diff
@@ -2911,8 +2911,11 @@ bulk_update_transfer_done_aux(const struct crt_bulk_cb_info *info)
 	return rc;
 
 send_error:
-	rc = crt_bulk_free(cb_info->buc_bulk_hdl);
 	output->rc = rc;
+	rc         = crt_bulk_free(cb_info->buc_bulk_hdl);
+	if (rc != 0)
+		DL_ERROR(rc, "crt_bulk_free() failed\n");
```

Comment:
Do you want to overwrite output->rc if it is 0?

Reply:
no, this is the 'send_error' path/label, and the rc returned from whatever failed above is the rc we want to send back. bulk_free() rc is only useful here for the subsequent DL_ERROR but nothing else